src/os/time/time.js

```diff
@@ -882,3 +882,22 @@ os.time.step = function(date, duration, offset, opt_local) {
   var next = os.time.offset(date, duration, offset, opt_local);
   return (next.getTime() - date.getTime());
 };
+
+
+/**
+ * Combines the days and time for seperate mapped fields
+ * @param {string} days The year, month, day
+ * @param {string} time The hours, minutes, seconds, milliseconds
+ * @param {string} daysFormat The format for days
+ * @param {string} timeFormat The format for time
+ * @return {moment}
+ */
+os.time.combineDateTime = function(days, time, daysFormat, timeFormat) {
+  var daysMoment = os.time.parseMoment(days, daysFormat, true);
```

Comment:
You should be able to avoid parsing into two separate `moment` objects by combining the format strings and parsing once. For example:

```
os.time.parseMoment(`${days} ${time}`, `${daysFormat} ${timeFormat}`, true)
```

This would be preferred in cases where this function may be called on a lot of data and performance is a concern.